docs/user-guide/src/SUMMARY.md

```diff
@@ -46,4 +45,5 @@
 - [Try it (for developers)](developer_environment/tryit.md)
 - [Version Support](version_support.md)
 - [Project Security Policy](security_policy.md)
+- [Troubleshooting FAQ](troubleshooting.md)
```

Comment:
It could be harder to discover the guide here, especially for people who're not interested in IPAM or even CAPM3. Maybe at least move it after IPAM and group the last 3 items into something like "For developers"?

Reply:
I moved it up now but didn't add the group for now. The thing is, if we add a group, that has to link to a proper page also or the build fails. So we would need an "intro page" for that group. I'm also a bit hesitant to "hide" the version support and security policy in that way since they may well be of interest to users. Finally, if we do the group, it would be logical to also change the path (e.g. `/security_policy.md` -> `development/security_policy.md`) and I believe we have links to the policy from practically every repo in the organization :grimacing:

Reply:
Don't hide security policy, it must stay on top level.